Src/Fido2/Objects/CredentialPublicKey.cs

```diff
@@ -93,7 +93,7 @@ public bool Verify(ReadOnlySpan<byte> data, ReadOnlySpan<byte> signature)
         throw new InvalidOperationException($"Missing or unknown kty {_type}");
     }
 
-    internal RSA CreateRsa()
+    internal RSA CreateRSA()
```

Comment:
This is a specific case for breaking the .NET naming rules, since the RSA class name is also capitalized.